templates/galaxy/config/job_conf.yml

```diff
@@ -169,18 +169,6 @@ galaxy_jobconf:
       params:
         amqp_url: "pyamqp://galaxy_cz01:{{ rabbitmq_password_galaxy_cz01 }}@mq.galaxyproject.eu:5671//pulsar/galaxy_cz01?ssl=1"
         galaxy_url: "https://usegalaxy.eu"
-        manager: production
```

Comment:
Maybe the job failed because `manager` was no longer specified.

Reply:
if not specified, _default_ should be used which I have specified on pulsar.